queries/highlights.scm

```diff
@@ -32,6 +32,11 @@
   "endfunction"
 ] @keyword.function
 
+[
+ "break"
+ "continue"
+] @keyword
+
```

Comment:
I think those two are something like `keyword.loop` or osmething like this.

In any case, can you add a test case for that in the highlights tests ?

Reply:
`@keyword.loop` is not on the list of highlight captures defined in [nvim-treesitter's CONTRIBUTING.md](https://github.com/nvim-treesitter/nvim-treesitter/blob/6f1bf2feec45ff7c9d7844602fe175907b7db633/CONTRIBUTING.md#highlights) and none of the other captures listed seem to fit, hence why I used a mere `@keyword`.

Thanks for reminding me about the highlighting tests; I'm working on it right now.

Reply:
Could you merge that part of the query with the other keyword list then? Thanks again

Reply:
Done.